iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -824,6 +824,13 @@ default Stream<Connectable> getConnectableStream() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Get a connectable by its ID or alias
+     *
+     * @param id the id or an alias of the equipment
+     */
+    Connectable getConnectable(String id);
```

Comment:
To prevent breaking the API, you can add a default implementation
```java
default Connectable<?> getConnectable(String id) {
       Identifiable<?> identifiable = getIdentifiable(id);
       if (identifiable instanceof Connectable<?>) {
             return (Connectable<?>) identifiable;
       }
       return null;
}
```

Reply:
(I know the wildcard is considered as a code smell, but I prefer you use it rather than nothing)